co_sim_io/co_sim_io.hpp

```diff
@@ -43,59 +43,51 @@ namespace CoSimIO {
 inline ReturnInfo Hello();
 
 
-inline ReturnInfo Connect(const std::string& rConnectionName, CoSimIO::SettingsType Settings);
-inline ReturnInfo Connect(const std::string& rConnectionName, const std::string& rSettingsFileName);
+inline ReturnInfo Connect(const ConnectionSettings& I_Settings);
 
-inline ReturnInfo Disconnect(const std::string& rConnectionName);
+inline ReturnInfo Disconnect(const Info& I_Info);
 
 
 template<class TContainerType>
 inline ReturnInfo ImportData(
-    const std::string& rConnectionName,
-    const std::string& rIdentifier,
+    const Info& I_Info,
```

Comment:
@pooyan-dadvand for all of those functions the minimum is the things I removed as separate arguments
I.e. the `connection_name` and the `identifier`
esp for the second one I am open for suggestions

Reply:
I agree!